frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/InboxSearchComposer.js

```diff
@@ -69,7 +81,7 @@ const InboxSearchComposer = (props) => {
                             <SearchComponent 
                                 uiConfig={ configs?.sections?.search?.uiConfig} 
                                 header={configs?.sections?.search?.label} 
-                                screenType={configs?.type}/>
+                                screenType={"search"}/>
```

Comment:
Why are we passing hard coded screen type for Search Component? because it will break css which is handled based on screen type